src/askem_beaker/contexts/mira_model_edit/procedures/python3/add_controlled_conversion_template.py

```diff
@@ -33,7 +33,7 @@
         subject = subject_concept,
         outcome = outcome_concept,
         controller = controller_concept,
-        rate_law = sympy.parsing.sympy_parser.parse_expr("{{ template_expression }}", local_dict=_clash),
+        rate_law = safe_parse_expr("{{ template_expression }}", local_dict=_clash),
```

Comment:
Where is `safe_parse_expr` defined. I can't find it in the beaker code or the mira library.

Reply:
It is defined here:

https://github.com/gyorilab/mira/blob/090deb783b77d6a841a94797fda40c237dcf2dfd/mira/metamodel/utils.py#L32